From 027229d25392b22d7280c0abbc3efde4f467d167 Mon Sep 17 00:00:00 2001
From: Peiran Hong <peiran.hong@windriver.com>
Date: Thu, 5 Sep 2019 15:42:22 -0400
Subject: [PATCH] Skip strip-trailing-cr test case

Skip the test "strip-trailing-cr" since it requires valgrind to
work, but valgrind is considered too heavy-weight for diffutils
package.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Peiran Hong <peiran.hong@windriver.com>

---
 tests/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index d98df82..757ea52 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -21,9 +21,11 @@ TESTS = \
   stdin \
   strcoll-0-names \
   filename-quoting \
-  strip-trailing-cr \
   timezone \
   colors
+# Skipping this test since it requires valgrind
+# and thus is too heavy for diffutils package
+# strip-trailing-cr
 
 XFAIL_TESTS = large-subopt
 
